### DISCUSSION TITLE
Forbidden and Arcanus

### DISCUSSION THREAD

**Discussion by TheFastHero** [DynamicTreesTeam/DynamicTrees#992] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/forbidden-arcanus

2 Trees

Version 1.20.1

**Comment by Groupix05** [comment]
https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-forbidden-arcanus (already in 1.12.2 and 1.16.5)

**Reply by TheFastHero** [reply]
Yes! I'm interested to see it ported to 1.20.1

**Comment by supermassimo** [comment]
Done https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-forbidden-arcanus
